songbird/helpers.py

```diff
@@ -1,14 +1,17 @@
-"""I use Arch btw so this module exists to remove bloat"""
+"""Static help functions."""
+
+# FIXME: Typing
+
+import typing
```

Comment:
Only form imports are used in this project

Reply:
What's wrong?

Reply:
Its typing